### PR TITLE
Resolve dead links

### PR DESCRIPTION
**Pull request by Gijsreyn**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

This PR is a minor change to docs for dead links, Markdown linting.
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/199841)

### CODE REVIEW

**Comment by Gijsreyn** [comment]
@denelon Same for this one :)

**Review by denelon — DISMISSED** [review]

**Comment by denelon** [comment]
You rock!

File: doc/manifest/schema/1.0.0/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [versionSchema]:        https://github.com/microsoft/winget-cli/blob/master/schemas/JSON/manifests/v1.0.0/manifest.version.1.0.0.json
 [defaultLocaleSchema]:  https://github.com/microsoft/winget-cli/blob/master/schemas/JSON/manifests/v1.0.0/manifest.defaultLocale.1.0.0.json
 [localeSchema]:         https://github.com/microsoft/winget-cli/blob/master/schemas/JSON/manifests/v1.0.0/manifest.locale.1.0.0.json
-[installerSchema]:      https://github.com/microsoft/winget-cli/blob/master/schemas/JSON/manifests/v1.1.0/manifest.installer.1.0.0.json
+[installerSchema]:      https://github.com/microsoft/winget-cli/blob/master/schemas/JSON/manifests/v1.1.0/manifest.installer.1.1.0.json
```
Comment (aisgbnok) [inline doc/manifest/schema/1.0.0/README.md:4]:
```suggestion
[installerSchema]:      https://github.com/microsoft/winget-cli/blob/master/schemas/JSON/manifests/v1.0.0/manifest.installer.1.0.0.json
```

Comment (Gijsreyn) [reply doc/manifest/schema/1.0.0/README.md]: Oops, I assumed the previous commitment to point it toward 1.1.0. Thanks for the sharp eyes, @aisgbnok

Comment (aisgbnok) [reply doc/manifest/schema/1.0.0/README.md]: Ty.

**Comment by denelon** [comment]
Great catch @aisgbnok

**Review by denelon — APPROVED** [review]

**Pull request merged**